Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -629,6 +629,8 @@ NonnullRefPtr<GUI::Action> HackStudioWidget::create_save_as_action()
         auto new_project_file = m_project->get_file(save_path.value());
         m_open_files.set(save_path.value(), *new_project_file);
         m_open_files_vector.append(save_path.value());
+
+        window()->set_title(String::formatted("{} - {} - Hack Studio", m_current_editor_wrapper->filename_label().text(), m_project->name()));
```

Comment:
Let's extract this line to a method (for example `update_window_tilte()`) to avoid duplicating the string format with what we already have in `set_current_editor_wrapper`.

Reply:
Done!

Reply:
Please squash this fix into one of the earlier commits :)

Reply:
oh okay, it should be fine now. :)